Lib/_pyio.py

```diff
@@ -588,10 +588,7 @@ def read(self, size=-1):
     def readall(self):
         """Read until EOF, using multiple read() call."""
         res = bytearray()
-        while True:
-            data = self.read(DEFAULT_BUFFER_SIZE)
-            if not data:
-                break
+        while (data := self.read(DEFAULT_BUFFER_SIZE)):
```

Comment:
Why are you using parentheses here? I don't think they are syntactically necessary.

Reply:
I'm too lazy to read the latest PEP, and the current implementation of the PEP is outdated, so I added parenthesis even if they are not needed.